contracts/ReputationMiningCycle.sol

```diff
@@ -35,6 +39,13 @@ contract ReputationMiningCycle {
     uint256 nNodes;
     uint256 lastResponseTimestamp;
     uint256 challengeStepCompleted;
+    bytes32 jrh;
```

Comment:
It's probably time to comment this struct members. Most of its members are non obvious in meaning